src/components/RegistrationPeriod/index.tsx

```diff
@@ -144,7 +164,10 @@ const EditRegPeriod = observer((props: Props) => {
                             <div className={clsx('card')}>
                                 <div className={clsx('card__header')}>
                                     <h4>
-                                        <Translate id="RegistrationPeriod.datePeriod.FromSemester">
+                                        <Translate
+                                            id="RegistrationPeriod.datePeriod.FromSemester"
+                                            description="RegistrationPeriod.datePeriod.FromSemester"
```

Comment:
in my opinion the `description` has no value, if it is the same as the `id` and can therefore be skipped.